src/Core/src/Platform/Android/MauiAppCompatActivity.cs

```diff
@@ -29,25 +29,24 @@ protected override void OnCreate(Bundle? savedInstanceState)
 			if (mauiApp.Services == null)
 				throw new InvalidOperationException("App was not initialized");
 
-			var window = mauiApp.GetWindowFor(null!);
+			var mauiContext = new HandlersContext(mauiApp.Services, this);
+			var window = mauiApp.CreateWindowFor(new ActivationState(savedInstanceState, mauiContext));
 
-			window.MauiContext = new HandlersContext(mauiApp.Services, this);
+			window.MauiContext = mauiContext;
```

Comment:
As noted in my comment about the Ini, if this is just a simple wrap, then maybe we can do it for the users coming in from the old way?

More importantly, the `HandlersContext` needs to be renamed `MauiContext` to match iOS and the filename and the base interface.

Reply:
> As noted in my comment about the Ini, if this is just a simple wrap, then maybe we can do it for the users coming in from the old way?

Once we have more defined windows/app/builder classes build then I think that will address it for users coming from old way.

I imagine the instructions will go
- Make your currrent "FormsAppCompatActibvity" inherit from MauiAppCompatActivity
- Delete all our code
- If you were Initing other third party libraries then you can go to the builder and do it like this... Or you can still do it here